design/design-doc-stacks.md

```diff
@@ -103,32 +109,32 @@ status:
 
 ## `Extension` CRD
 
-The `Extension` CRD serves as a record of an installed extension (a custom controller and its CRDs).
+The `Extension` CRD serves as a record of an installed Stack (a custom controller and its CRDs).
 These records make it so that a user or system can query Crossplane and learn all of the functionality that has been installed on it as well as their statuses.
 
 Instances of this CRD can be generated from the filesystem based contents of a package, i.e. the metadata files contained inside the package.
 This can be thought of as a translation operation, where the file based content is translated into a YAML based version that is stored in the `Extension` CRD.
 
 `Extension` CRD instances can also be created directly by a user without any knowledge of packages at all.
-They can directly create any CRDs that their extension requires and then create an `Extension` CRD instance that describes their extension, its custom controller, etc.
-The Extension Manager will see this new instance and take the steps necessary to ensure the custom controller is running in the cluster and the extension’s functionality is available.
+They can directly create any CRDs that their Stack requires and then create an `Extension` CRD instance that describes their Stack, its custom controller, etc.
+The Stack Manager will see this new instance and take the steps necessary to ensure the custom controller is running in the cluster and the Stack’s functionality is available.
 
 ```yaml
 apiVersion: extensions.crossplane.io/v1alpha1
 kind: Extension
```

Comment:
Without much context on this change, `Extension.stack.crossplane.io` and `ExtensionRequest.stack.crossplane.io` may be good names if we decide to rename the CRDs. Similar to how the “workload” functionality is implemented using `KubernetesApplication.workload.crossplane.io`

Reply:
I'll keep this in mind when creating (or reviewing) the follow-up PR to rename the code.